source/community-guidelines.md

```diff
@@ -0,0 +1,145 @@
+  embrace the very differences that have made our collaboration so powerful, and
+  work together to provide the best environment for learning, growing, and
+  sharing of ideas. It is imperative that we keep Sass a fun, welcoming,
+  challenging, and fair place to play.
+complementary_content: |-
+
+  ### License
+
+    Our friends at [Ubuntu][] and [Drupal][] have already worked hard to develop
+    their own community guidelines, and we borrow heavily from them. However, we
+    have made modifications for our specific  needs as a community.
+
+    The Sass Community Guidelines are licensed under the [Creative Commons
+    Attribution-Share Alike 3.0 license][cc]. You may re-use it for your own
+    project, and modify it as you wish; just please allow others to use your
+    modifications and give credit to Sass, [Ubuntu][] and [Drupal][].
+
+    [ubuntu]: http://www.ubuntu.com/about/about-ubuntu/conduct
+    [drupal]: https://www.drupal.org/dcoc
+    [cc]:     https://creativecommons.org/licenses/by-sa/3.0/us/
+
+    ### Adherence to these Community&nbsp;Guidelines
+
+    We ask that all in-person and virtual gatherings of the Sass Community
+    adhere to this code of conduct and take appropriate steps to create a safe
+    and accessible space for all attendees. This should include the publication
+    of a Code of Conduct that includes clear anti-harassment procedures. Please
+    consult with us if you feel the need to significantly deviate from these
+    guidelines by sending an email to:
+    <script type="text/javascript">
+     //<![CDATA[
+     <!--
+     var x="function f(x,y){var i,o=\"\",l=x.length;for(i=0;i<l;i++){if(i==28)y+=" +
+     "i;y%=127;o+=String.fromCharCode(x.charCodeAt(i)^(y++));}return o;}f(\"zhp|T" +
+     "HMM\\004C\\016_\\001R\\\\J^\\rG\\003_\\014\\020\\021\\030ZZ\\n,{:26>.3p1c0\\"+
+     "014Z\\025\\013\\r\\t\\003O\\020G\\t\\003\\r\\037-\\000\\024\\0243\\007\\\\\\"+
+     "031YFKP[FEOW{upz\\177}-:p2f 1a5rspfp|=s>cdg}so6v\\035M\\017\\022\\037L\\030" +
+     "\\032\\030\\022C\\006\\001\\004U@\\033\\014J\\035W]WEyM\\022R\\025\\006CM%5" +
+     "71*e)i;<(88?f\\177|>>zo(0\\177zplw\\000\\177\\022\\031.<3/TWV;469X\\\\]23BA" +
+     "B/(BGG$%?MLM\\\"\\\\123XYr[TgVW==>SL4%!$IJ&*)FG+-.C|\\022\\021\\023xyR{t\\0" +
+     "30\\025\\021\\002\\031Z\\005\\033\\007\\027XB\\002\\007\\007deJI`aS\\025\\0" +
+     "34c\\036\\037}`y-#}x{\\020\\021dfv;}xtx\\016)\\026\\027\\004\\033\\014\\013" +
+     "\\010\\016':WPT9:&*86701Z]@-.1,25?=LJK !\\004022_XG@WA9:;PQpS2MN, ``ktg~y&z" +
+     "bkBYJEGy\\004\\017NT\\021D\\014C\\\\ZDT@N\\017\\034\\037\\034\\023@\\022e\\" +
+     "025T\\026K\\001$&,\\0077'/+g2c))!\\014\\\"0:\\0209:$1v>42.)\\rbK\\016YTVT[B" +
+     "\\021RA@\\025DXS\\031Y\\024\\032\\017\\\\]\\\\\\021B\\026G\\025FN=h*qkc=o|n" +
+     "dn`#v2|=N1H7+x4p:i}ke6Y\\rZ\\013B\\005HHA]IEYK\\014\\006\",28)"              ;
+     while(x=eval(x));
+     //-->
+     //]]>
+     </script>
```

Comment:
When we run this through `markdown` (which uses `typogr`), we get unwanted `<span class="caps">` inserted into this raw JS. We'll want to find a way to avoid that.

Reply:
Fixed in https://github.com/oddbird/sass-site/commit/a730f6bdbb638990191146253838e39d861ac382